src/test/java/ca/corefacility/bioinformatics/irida/ria/integration/pages/announcements/AnnouncementControlPage.java

```diff
@@ -79,10 +80,11 @@ public void clickCreateNewAnnouncementButton() {
     }
 
     public void gotoMessageDetails(int index) {
-        List<WebElement> messages = driver.findElements(By.cssSelector("td.t-announcement a"));
+        List<WebElement> messages = driver.findElements(By.cssSelector("td.t-announcement p"));
```

Comment:
This change was pulled in from https://github.com/phac-nml/irida/pull/699 as announcements page tests were failing.